### PR TITLE
Temporarily increase Freesound delay & timeout

### LINKED ISSUE #932: Temporarily increase the delay & timeout for Freesound
## Description

Before converting Freesound to a dated DAG, we are trying to complete one full run (#931). 

There is a [2000 request per day rate limit](https://freesound.org/docs/api/overview.html#throttling). In order to respect this, and hopefully to help circumvent some flakiness we've seen with API responses, we need to significantly increase the delay between requests. Note that we currently make requests for each page of 150 results, as well as additional requests for audio_set information and file size.

We should also increase the `pull_timeout` significantly to ensure that the full DAG can run with the increased delay.

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Fixes

Fixes #932 by @stacimc

## Description

This PR temporarily increases the API request delay from the default (1 second) to 5 seconds. It also changes the `pull_timeout` from the default (24 hours) to 48 hours.

## Testing Instructions

The pull timeout change should be shown on the task instance detail page for Freesound's `pull_media` task (see `execution_timeout`). Although if you _do_ run it locally you may also run into #659.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: openverse_catalog/dags/providers/provider_api_scripts/freesound.py [hunk 1]
```diff
@@ -6,8 +6,10 @@
 Output:                 TSV file containing the image, the respective
                         meta-data.
 
-Notes:                  https://freesound.org/apiv2/search/text'
-                        No rate limit specified.
+Notes:                  https://freesound.org/apiv2/search/text.
+                        Rate limit is specified here (2k requests/day):
+                        https://freesound.org/docs/api/overview.html#throttling.
+                        Maximum page size is 150.
```
Comment (krysal) [inline openverse_catalog/dags/providers/provider_api_scripts/freesound.py:12]: Can we undo this change to avoid conflict with #928?

**Review by krysal — APPROVED** [review]
LGTM. Just left a minor request to undo the comment. Thanks!

File: openverse_catalog/dags/providers/provider_workflows.py [hunk 2]
```diff
@@ -137,6 +137,7 @@ def __post_init__(self):
     ),
     ProviderWorkflow(
         ingester_class=FreesoundDataIngester,
+        pull_timeout=timedelta(hours=48),  # TODO: Remove after full Freesound run
```
Comment (stacimc) [inline openverse_catalog/dags/providers/provider_workflows.py:140]: I'm not sure 48 hours will be sufficient 🤔 The 2022-11-01 run in production (the `skip_ingestion_errors` run) timed out after 24 hours with the delay at 1 second. Can we extend this further? I think it could realistically take a week, maybe two to be safe as long as we check in on it regularly.

File: openverse_catalog/dags/providers/provider_api_scripts/freesound.py (hunk 1 above)
Comment (AetherUnbound) [reply openverse_catalog/dags/providers/provider_api_scripts/freesound.py:12]: Oh shoot, yes! Good callout!

File: openverse_catalog/dags/providers/provider_workflows.py (hunk 2 above)
Comment (AetherUnbound) [reply openverse_catalog/dags/providers/provider_workflows.py:140]: Ahh, thanks for that extra context! Yea I think 240 hours (10 days) would be a good place to start then!

**Review by stacimc — APPROVED** [review]
LGTM!
